py3status/modules/spaceapi.py

```diff
@@ -57,10 +52,10 @@ def check(self):
                 self.closed_color = ''
 
             # grab json file
-            json_file = urllib.request.urlopen(self.url)
-            reader = codecs.getreader("utf-8")
-            data = json.load(reader(json_file))
-            json_file.close()
+            try:
+                data = self.py3.get_json_data(self.url)
+            except self.py3.Exceptions:
+                self.py3.notify_user('{} cannot be reached.'.format(self.url))
```

Comment:
This should be updated we don't get an exception and need to handle the None case